ModelFarm/src/test/java/io/deephaven/modelfarm/TestModelFarm.java

```diff
@@ -22,7 +22,7 @@
 public class TestModelFarm extends TestCase {
 
     private final long testShutdownTimeoutSecs =
-            Configuration.getInstance().getIntegerWithDefault("TestModelFarm.testShutdownTimeoutSecs", 1);
+            Configuration.getInstance().getIntegerWithDefault("TestModelFarm.testShutdownTimeoutSecs", 5);
```

Comment:
Were you planning to commit a bigger default? Did CI in fact time out, or was something else wrong?

Reply:
I wasn't planning to increase the constant.  The bug was a branch that wasn't getting handled.  I just ran with the old constant and had zero failures for the first two (faster) test cases in >50k tries.